client/src/api/user/index.ts

```diff
@@ -149,6 +154,15 @@ const declineUser = async (
   return await put(url, onboardingStatusEnum.DENIED, failureMessage);
 };
 
+const getAggregatedUser = async (
```

Comment:
This isn't something for you to change, but I am almost wondering if it would have been easier to just have a Promise.all at the beginning of these components that called all of the information that that page needs rather than doing this aggregate thing I came up with. I think it is leading to a lot of these issues with null errors. Just thinking a loud though dw about doing anything with this.

Reply:
I think both work honestly, but I'm a fan of the aggregation since it keeps all the data relationships intact.